squape/settings.py

```diff
@@ -0,0 +1,265 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2023, Cyber Alpaca
+# All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+from contextlib import contextmanager
+
+import squish
+
+
+@contextmanager
+def _ctx_settings(setting_name, value):
+
+    try:
+        current_value = getattr(squish.testSettings, setting_name)
+        setattr(squish.testSettings, setting_name, value)
+    except AttributeError as err:
+        raise AttributeError(
+            f"Your Squish version does not support test setting {setting_name}\n{err}"
+        )
```

Comment:
Could you create a dedicated Exception type that could be used by other modules in the future as well?

Reply:
Added custom SquishCapability exception